tox.ini

```diff
@@ -1,8 +1,7 @@
 [tox]
-envlist = py3, pep8
+envlist = py3, py35, py36, pep8
 
```

Comment:
> Given that py3 is a default environment can't we omit it ?

Oh, you are right, it seems like "py3" is understood by tox by default. I wasn't aware of them.

Ok, in this case only pep8 should force basepython=python3.